features/gezag-minderjarige/4a.2-ouders-overleden-of-onbevoegd-tot-gezag.feature

```diff
@@ -0,0 +1,456 @@
+  
+  Mogelijke uitkomsten:
+  1) Nee
+  -> OG2: Beide ouders hebben gezamenlijk het ouderlijk gezag
+  2) Ja, één van beide ouders
+  -> OG1: De andere ouder heeft alleen het gezag (1:253f BW bij overlijden of 1:253q lid 1 BW bij onbevoegdheid, of art 251b BW)
+  3) Ja, beide ouders
+  -> G: Er is (tijdelijk) geen gezag (1:253g lid 1 BW bij overlijden of 1:253q lid 2 BW bij onbevoegdheid)
+
+  Op het moment dat de moeder weer bevoegd wordt tot het gezag, verkrijgt zij van rechtswege het gezag, tenzij op dat tijdstip een ander met het
+  gezag is belast (art. 1:253b lid 2 BW). Dit geldt ook o.g.v. art. 1:253q lid 1 BW.
+  Let op: De kinderrechter kan een minderjarige moeder o.b.v. artikel 1:253ha lid 1 BW meerderjarig verklaren. Dit is nu niet in de BRP te achterhalen,
+  dit is wel zichtbaar als aantekening in het Gezagsregister.
+
+  Hoe achterhalen:
+  Niet overleden: op de persoonslijst van de ouder(s) mag de datum opschorting bijhouding niet voorkomen.
+  Meerderjarig: de geboortedatum van de ouder(s) moet 18 jaar of hoger zijn.
+  Niet onder curatele: op de persoonslijst van de ouder(s) mag de indicatie curatele niet voorkomen.
+
+  Het wordt in de BRP niet geregistreerd dat de geestelijke vermogens van een ouder gestoord zijn.
+  Uitzondering genoemd in de wet Gezamenlijk gezag door erkenning: De moeder en de persoon die een kind heeft erkend oefenen het gezag over hun
+  kind gezamenlijk uit, tenzij
+  a. een voogd met de voogdij over het kind is belast;
+  b. de voorziening in het gezag over het kind is komen te ontbreken; of
+  c. de ouder die het gezag heeft, dit gezamenlijk met een ander dan een ouder uitoefent.
+
+  De moeder oefent het gezag over haar kind alleen uit, indien uit de akte, bedoeld in artikel 203, eerste lid, onderdeel a of b, blijkt dat:
+  a. de moeder en de erkenner hebben verklaard dat het gezag alleen door de moeder wordt uitgeoefend; of
+  b. de rechtbank vervangende toestemming voor de erkenning heeft verleend als bedoeld in artikel 204, derde en vierde lid.
+  De ambtenaar van de burgerlijke stand zendt onverwijld een afschrift of uittreksel van de akte aan de griffier ter aantekening van de verklaring,
+  bedoeld onder a, of de erkenning met vervangende toestemming, bedoeld onder b, in het register, bedoeld in artikel 244.
+
+  Opmerkingen
+  • Vraag 4a.2: Het antwoord op de vraag 4a.2 is mogelijk te beperkt om gezag juist te bepalen. De gevolgen van overlijden en curatele
+    voor gezag zijn verschillend. Of komt de situatie van curatele eigenlijk niet voor omdat er dan al een uitspraak gezag voorligt?
+
+  Aanbeveling
+  • Meerderjarig verklaring: ophalen meerderjarig verklaring uit de aantekening in het Gezagsregister
+  • Mogelijk moet autorisatie worden verleend voor Beschrijving Document vanwege de instructie opnemen “van rechtswege, erkenning OV 2022”.
+
+ TODO: Verplaatsen naar developer testen - Gebruikte velden:
```

Comment:
Er staat hier nog een TODO in. Voorstel om hier een issue van te maken voor het opnemen van developer testen

Reply:
Zoals besproken TODO wegehalen en andere vorm laten staan